docs/index.md

```diff
@@ -4,7 +4,141 @@ hide:
   - toc
 ---
 
-<h1 align="center" style="margin-top: 0px;">Dasharo OSF vPub Summer 2022</h1>
-<h2 align="center" style="margin-top: 0px;">More info coming soon!</h2>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
-If you are interested in previous events please visit our [archive](archive.md).
+<p align="center" style="margin-bottom: 0px !important;">
+  <img width="800" src="../images/hackathon-2022-poster.png" alt="Hackathon 2022 poster" align="center">
```

Comment:
Poster should also contain Dasharo branding and whole event should promote Dasharo

Reply:
Maybe call it coreboot flashing party or something like that. And with small letters that it would be in form of workshop with elements of hackathon.